circus-train-api/src/test/java/com/hotels/bdp/circustrain/api/conf/TableReplicationTest.java

```diff
@@ -89,4 +93,26 @@ public void nullTableLocation() throws Exception {
     assertThat(violations.size(), is(1));
   }
 
+  @Test
+  public void mergeNullOptions() {
+    Map<String, Object> mergedCopierOptions = tableReplication.getMergedCopierOptions(null);
+    assertThat(mergedCopierOptions, is(not(nullValue())));
```

Comment:
A general question: is there a reason to prefer the generic `assertThat` to more specific methods, like `assertNotNull` in this case?

Reply:
We prefer hamcrest. I'm ambivalent either way. This was copy paste since the code was moved I moved the test code as well.